hack/setup_sonobuoy.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function is_kind_available() {
+  if ! command -v kind &> /dev/null
+  then
+    echo "kind could not be found"
+    exit
+  fi
+}
+
+function setup_sonobuoy() {
+  if [ -f ./sonobuoy/sonobuoy ]; then
+    echo "Found Sonobuoy, skipping setup."
+  else 
```

Comment:
``` if ... then
      echo "Found Sonobuoy, skipping setup."
      exit 1
   fi
```

Reply:
will do this in a follow on , lets merge and iterate to get this in before the mtng today

Reply:
okay, @jayunit100. @nehaLohia27 could you please fix this in your further work on sonobuoy?

Reply:
yup merged making pr now

Reply:
ah wait i did e 1 and that fails for other reasons bc i have sonobuoy already